.github/workflows/dev-fe-e2e.yaml

```diff
@@ -149,7 +149,7 @@ jobs:
           MONITORING_PASSWORD: "admin"
         run: |
           cd ui
-          pnpm --filter "@percona/everest" e2e
+          pnpm --filter "@percona/everest" e2e --grep-invert @release
```

Comment:
Thinking about how to trigger the `@release` tests then. 
Triggering them by a new tag is not correct, bc the build does not exist at that moment. 
Looks like [the release trigger](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#release) would be relevant:
```
on:
  release:
    types: [published]
```
It should work for both prereleases (RCs) and releases. And then we can use the appropriate build to set up Everest.

Reply:
I was thinking for these to just build the cli and use the current dev latest build and if the release branch exists to be able to point it to the release branch and use already pre-built cli and package.
But these will likely be executed as well on different platforms and will need some credentials for cloud providers so these pipelines will be added to private [k8s-testing-automation](https://github.com/percona/k8s-testing-automation) repo.